jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(30000)
```

Comment:
@shrijan-siris Is there a reason our tests need 30 seconds timeout? Ideally, it should be pretty quick.

Reply:
some times async function may need more time to finish